contents/newsletter/feature-flag-mistakes.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Feature flag mistakes to avoid
```

Comment:
Headline ideas:

- Don't make these classic feature flag mistakes
- Avoid these classic feature flag mistakes
- Feature flag mistakes that will cost you time and money

I don't love any of these, but it feels like this need something extra